CIP-0068/README.md

```diff
@@ -1,77 +1,111 @@
 ---
 CIP: 68
 Title: Datum Metadata Standard
-Authors: Alessandro Konrad <alessandro.konrad@live.de>, Thomas Vellekoop <thomas.vellekoop@iohk.io>
-Comments-URI:
 Status: Proposed
-Type: Informational
+Category: Meta
+Authors: 
+  - Alessandro Konrad <alessandro.konrad@live.de>
+  - Thomas Vellekoop <thomas.vellekoop@iohk.io>
+Implementors:
+Discussions:
```

Comment:
```suggestion
Discussions:
  - https://github.com/cardano-foundation/CIPs/pull/586/
  - 
```
Probably a couple PRs worth adding here

Reply:
resolving: dup of https://github.com/cardano-foundation/CIPs/pull/586#discussion_r1362429840